scripts/docker/supervisor.conf

```diff
@@ -18,3 +18,12 @@ port=9001
 
 [rpcinterface:supervisor]
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[program:celeryd]
+command=python manage.py celery worker --autoreload --broker=redis://redis:6379/1 -Q priority,devhub,images,limited,celery --loglevel DEBUG
```

Comment:
I'll remove the loglevel here, it was for testing. Although it *is* nice for logging. Hmm.